sql/moz-fx-data-shared-prod/telemetry_derived/newtab_desktop_daily_engagement_by_tile_id_position_v1/schema.yaml

```
@@ -0,0 +1,25 @@
+fields:
+- name: submission_timestamp
+  type: TIMESTAMP
+  mode: NULLABLE
```

I'm not sure if you want this as a date or aggregated by hour or what. Let me know if you want to talk through this

```suggestion
- name: submission_date
  type: DATE
  mode: NULLABLE
```